### DISCUSSION TITLE
Hover over panel and Cmd+D to split

### DISCUSSION THREAD

**Discussion by amacneil** [foxglove/community#83] [Q&A]
**Description**
I want to be able to easily split panels using keyboard shortcuts.

As a mac/iterm user, `Cmd+d` for for a vertical split and `Cmd+Shift+d` for a horizontal split would make the most sense. We should research common shortcuts on windows/linux.

I propose that we combine this split behavior with "focus follows mouse", so those keyboard shortcuts will split whichever pane my mouse is currently over.

cc @2metres @jtbandes @defunctzombie for thoughts

**Comment by rgov** [comment]
Note that <kbd>⌘⇧D</kbd> on macOS Terminal.app closes a (vertical) pane, it doesn't create a new one. (It is a feature I never use so I'm not personally offended if there's a conflict here.) I couldn't find a key combo for this in Xcode. Visual Studio Code seems to default to <kbd>⌘\\</kbd>.
